scipy/signal/tests/test_signaltools.py

```diff
@@ -1281,9 +1281,20 @@ def test_scale_invariance(self):
 
 
 class TestFiltFilt(TestCase):
+    filtfilt_kind = 'tf'
```

Comment:
It looks like this never gets set to `'sos'`, though the tests do pass with it manually switched on my machine.